src/adapter/http/src/data/router.rs

```diff
@@ -29,6 +29,18 @@ pub fn root_router() -> axum::Router {
             "/verify",
             axum::routing::post(super::verify_handler::verify_handler),
         )
+        .route(
+            "/workspace/info",
```

Comment:
Let's use `/info` as this will be returning very fundamental information about the ODF node